README.md

```diff
@@ -1,5 +1,7 @@
 # alias-lb
 
+[![Build Status](https://drone.techservices.illinois.edu/api/badges/techservicesillinois/terraform-aws-alias-lb/status.svg)](https://drone.techservices.illinois.edu/techservicesillinois/terraform-aws-alias-lb)
+
 Provides a Route 53 alias whose target is an Amazon application load balancer (ALB).
 
 This module provides a Route 53 alias whose target is an application load balancer. This module exists for cases in which a Route 53 alias needs to be managed using a _separate life cycle_ from the load balancer itself or an ECS service.
```

Comment:
Please combine the first sentence (line 5) with the next paragraph (line 7), and drop the first sentence of the paragraph as it is says the same thing as the first sentence (line 5). I recommend lines 5,6,7 be changed to:

```
Provides a Route 53 alias whose target is an Amazon application load balancer (ALB). This module exists for cases in which a Route 53 alias needs to be managed using a _separate life cycle_ from the load balancer itself or an ECS service.
```